news/feature-3438.md

```diff
@@ -0,0 +1 @@
+network: add FreeBSD support for the `so_reuseport(yes)` the same as in linux `SO_REUSEPORT` (FreeBSD uses `SO_REUSEPORT_LB` flag).
```

Comment:
Note:

`SO_REUSEPORT` on BSD works differently: it's for broadcast and multicast datagrams, hence the `SO_REUSEPORT_LB` flag:

> SO_REUSEPORT_LB allows completely duplicate bindings by multiple pro-
     cesses if they all	set SO_REUSEPORT_LB before binding the port.  Incoming
     TCP and UDP connections are distributed among the sharing processes based
     on	a hash function	of local port number, foreign IP address and port num-
     ber. A maximum of 256 processes can share one socket.

Reply:
A pretty neat link about differences: https://stackoverflow.com/questions/14388706/how-do-so-reuseaddr-and-so-reuseport-differ (with a little history in it).